queue_services/entity-emailer/tests/unit/email_processors/test_agm_location_change_notification.py

```diff
@@ -0,0 +1,54 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Unit Tests for AGM location change email processor."""
+from unittest.mock import patch
+
+import pytest
+from legal_api.models import Business
+
+from entity_emailer.email_processors import agm_location_change_notification
+from tests.unit import prep_agm_location_change_filing
+
+
+@pytest.mark.parametrize('status,legal_name,is_numbered', [
+    ('COMPLETED', 'test business', False),
+    ('COMPLETED', 'BC1234567', True),
+])
+def test_agm_location_change_notification(app, session, status, legal_name, is_numbered):
```

Comment:
Open to adding more test cases if deemed useful, I just am not sure which ones to add.

Reply:
can we add one to verify that no notification is sent out when agm location change with PAID status queue message is received.